code/LINDA/LINDA_turf_tile.dm

```diff
@@ -270,14 +270,10 @@ turf/simulated/proc/share_temperature_mutual_solid(turf/simulated/sharer, conduc
 
 
 atom/movable/var/pressure_resistance = 5
-atom/movable/var/last_forced_movement = 0
 
 atom/movable/proc/experience_pressure_difference(pressure_difference, direction)
-	if(last_forced_movement >= air_master.current_cycle)
```

Comment:
Again, I'd like you to test it out instead of just side-commenting on it without doing so first---as this isn't exactly something you can just look at and know the exact effects of.

Recall that you added an artificial cooldown to prevent that from happening...which was later removed...and it didn't happen--I didn't see it happen here, which is why I'd like you to test with the game actually, well, running.